[Bug] Kotlin: Can't delete file

```
C:\GitHub\Anki-Android-Git\Anki-Android>git status
On branch new-async
Your branch is up to date with 'origin/new-async'.

You are currently bisecting, started from branch '8951-rebase'.
  (use "git bisect reset" to get back to the original branch)

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        modified:   AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/AsyncLayer.kt
        deleted:    AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/SchedulerService.kt


C:\GitHub\Anki-Android-Git\Anki-Android>git commit -m "aaa"
Running Ktlint over these files:
AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/AsyncLayer.kt

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':AnkiDroid:runKtlintFormatOverMainSourceSet'.
> A failure occurred while executing org.jlleitschuh.gradle.ktlint.worker.KtLintWorkAction
   > C:\GitHub\Anki-Android-Git\Anki-Android\AnkiDroid\src\main\java\com\ichi2\anki\servicelayer\SchedulerService.kt (The system cannot find the file specified)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2s

C:\GitHub\Anki-Android-Git\Anki-Android>

```

--- a/pre-commit
+++ b/pre-commit
@@ -11,6 +11,8 @@ fi;
 echo "Running Ktlint over these files:"
 echo "$CHANGED_FILES"
 
+# Clean is required because of https://github.com/ankidroid/Anki-Android/issues/9521
+./gradlew clean
 # -q removes noise from the output if it fails.
 # TODO: -w is better, but https://github.com/JLLeitschuh/ktlint-gradle/issues/457 adds noise
 # -w should display: "CriticalExceptionTest.kt:19:1 Wildcard import (cannot be auto-corrected)"

